sqlmesh/core/analytics/__init__.py

```diff
@@ -2,7 +2,7 @@
 
 import atexit
 import os
-import typing as t
+import typing_extensions as t
```

Comment:
Should we put this and the two globals under a `t.TYPE_CHECKING` branch?

Reply:
We can. I changed it. Let me know if it's what you had in mind.